docs/web-apps/automated-testing/_partials/_advanced-cypress.md

```diff
@@ -3,12 +3,10 @@
 With [cypress-grep](https://github.com/cypress-io/cypress-grep) you can add tags to test descriptions, then filter your tests with the tags or keywords.
 
 ```javascript
-it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
-cy
-.get('.action-email')
-.type('fake@email.com')
-.should('have.value', 'fake@email.com')
-})
+	it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
+    cy.get('.action-email')
+        .type('fake@email.com').should('have.value', 'fake@email.com')
+  })
```

Comment:
I guess that should be more this way or something like this or similar.
Note: In current change, 1st blank (before `it` is a `<tab>`.

```suggestion
    it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
      cy.get('.action-email')
        .type('fake@email.com').should('have.value', 'fake@email.com')
    })
```